confd/tests/mock_data/calicoctl/explicit_peering/route_reflector_v6_by_ip/input.yaml

```diff
@@ -0,0 +1,93 @@
+kind: BGPConfiguration
+apiVersion: projectcalico.org/v3
+metadata:
+  name: default
+spec:
+  asNumber: 64532
+  nodeToNodeMeshEnabled: false
+
+---
+
+# This BGPPeer peers the RR node (kube-master) with an explicit
+# external peer.
+kind: BGPPeer
+apiVersion: projectcalico.org/v3
+metadata:
+  name: bgppeer-1
+spec:
+  peerIP: 172.19.4.87
+  asNumber: 64533
+  nodeSelector: has(routeReflector)
+
+---
+
+# This BGPPeer peers the RR node (kube-master) with an explicit
+# external v6 peer.
+kind: BGPPeer
+apiVersion: projectcalico.org/v3
+metadata:
+  name: bgppeer-v6
+spec:
+  peerIP: "[ac13::57]:50"
+  asNumber: 64533
+  nodeSelector: has(routeReflector)
+
+---
+
+# This BGPPeer peers the RR node (kube-master) with the other
+# non-RR nodes in the cluster (kube-node-1, kube-node-2) using
+# its IP address only; this should be picked up as a bidirectional
+# peering but over IPv6 only.
+kind: BGPPeer
+apiVersion: projectcalico.org/v3
+metadata:
+  name: bgppeer-other-nodes
+spec:
+  peerIP: "fe0a::2"
```

Comment:
New test is based on the old route_reflector test, with this change to the node-to-node peering.